cmd/main.go

```diff
@@ -96,7 +98,8 @@ func main() {
 	c.Flags().StringVar(&redisDatabase, "redis-database", "", "Redis database")
 	c.Flags().IntVar(&redisPort, "redis-port", 0, "Redis Port")
 
-	c.Flags().StringVar(&fflag, "feature-flag", "", "Enable feature flags (experimental)")
+	c.Flags().StringSliceVar(&fflag, "enable-feature-flag", []string{}, "List of feature flags to enable e.g. \"search_tokenizer,prometheus\"")
```

Comment:
@mekilis Can we change this from `search_tokenizer` to `webhook-full-text-search`.

Reply:
Ok. Should we use `-` or `_` as the delimiter? I prefer `-`.